MFVNeutralino/python/VertexHistos_cfi.py

```diff
@@ -4,6 +4,7 @@
                                  mevent_src = cms.InputTag('mfvEvent'),
                                  weight_src = cms.InputTag('mfvWeight'),
                                  vertex_src = cms.InputTag('mfvSelectedVerticesTight'),
+                                 gen_particles_src = cms.InputTag('prunedGenParticles'),
```

Comment:
@abbywarden @pkotamnives are we sure about this one? I am finding that I cannot run VertexHistos for background and most (but oddly not all) signal samples, and it seems to be getting stuck here. I could remove this locally and then comment out all of the lines in VertexHistos.cc that relate to gen-level particles, but could there be something else that either didn't get committed or that I am misunderstanding?

Reply:
I could conceivably change ntuple.py to set `keep_gen = True` (and remake the ntuples) but even then, it would fail for data, which we'll want to look at in the not-too-distant-future! As it stands, I don't think it will work in data, so we should at minimum add some switches to protect it from crashing when getting to this point without having that collection available.